README.md

```diff
@@ -1,6 +1,7 @@
 # Hello-React-frontend-Project
 This application is developed using React and Redux.
 #### Link to Rails App [Hello-rails-backend](https://github.com/Tadessejemal1/Hello-rails-backend.git)
```

Comment:
- Kindly add the link of the backend  pr to the frontend readme, so the next reviewer can easily navigate to the pr 👍🏿